data/lhc-hxswg/README.md

```
@@ -12,6 +12,7 @@ Announced in HN: https://hypernews.cern.ch/HyperNews/CMS/get/higgs/1738.html
 - The ggH SM x-sections are calculated at N3LO, in these files, the uncertainty "Gauss" should be used for the scale uncertainty 
 - The ZH cross-section include the contribution from ggZH but ggZH is also provided separately so that a subtraction of ZH-ggZH will give the normalisation of qqZH 
 - There are numbers provided separately for the cross-sections of (W+)H and (W-)H in addition to the sum of the two (these are the LAST two columns inside 13TeV-WH.txt).
-- Separate numbers for PDF and alpha_S are also provided:
-- The tHW cross-sections given are repeated from the value at 125.0 GeV. This is only to allow bulding a spline.
+- The tHW cross-sections given are repeated from the value at 125.0 GeV. This is only to allow bulding a spline. These numbers are from YR4
 - The cross-sections and branching ratios are also provided inside `RooWorkspaces` as splines for your convenience as a function of `MH`. You can use these inside your datacards using `rateParams` to normalise the different Higgs processes. An example datacard is provided in `data/tutorials/rate_params/simple_sm_datacard.txt`.
+
+Please see the full set of notes from [here](https://gitlab.cern.ch/LHCHIGGSXS/LHCHXSWG1/crosssections) for all details.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)